py/dml/objects.py

```diff
@@ -256,8 +256,8 @@ def logname_anonymized(self, indices=(), relative='device'):
         if self.isindexed():
             suff = "".join('[%s]' % i for i in
                            indices[-self.local_dimensions():])
-            suff += "".join('[$%s]' % idxvar for idxvar in
-                            self._idxvars[len(indices):])
+            suff += "".join(f'[{dollar(self.site)}{idxvar}]'
+                            for idxvar in self._idxvars[len(indices):])
```

Comment:
This _is_ breaking! `logname_anonymized` is used unsaturated in order to generate event class names! That enough is a reason to revert this change.

Reply:
Reverted.